opdracht1.js

```diff
@@ -0,0 +1,23 @@
+// opdracht 1A
+let tvTypes = inventory.map((types) =>{
+    return types.type
+})
+console.log(tvTypes)
+
+//opdracht 1B
+let soldOutTelevisions = inventory.filter((actualStock) =>{
+    return actualStock.sold >= actualStock.originalStock
+})
+console.log(soldOutTelevisions)
+
+//opdracht 1C
+const ambiLightTvs = inventory.filter((alTv) =>{
+    return alTv.options.ambiLight === true
+})
```

Comment:
Deze zou je nog wat korter kunnen schrijven
```suggestion
const ambiLightTvs = inventory.filter(alTv => alTv.options.ambiLight);
```